src/cmdlib.sh

```diff
@@ -278,14 +278,52 @@ runvm() {
     # include COSA in the image
     find /usr/lib/coreos-assembler/ -type f > "${vmpreparedir}/hostfiles"
 
+    # virtfs contains additional 9p mounts, when needed
+    virtfs=()
+
+    # Allow run-time defined config files (e.g. registry credentials)
+    # Use NAME=FILE, which are exposed as envVars NAME=<INSIDE LOCATION>.
+    # The reason for copying these files is that outside path
+    # is likely not accessible inside the VM or they have prohibited permissions,
+    # as in the case of Openshift mounted secrets.
+    local extras
+    if [ "${RUNVM_CONFIG_FILES}x" != "x" ]; then
```

Comment:
This seems OK to me, though it feels like it might be nicer if we *required* the config files to be in a certain place and always copied them in if present.

E.g. rather than specifying `RUNVM_CONFIG_FILES=/run/containers/.../auth.json` we just hardcoded copying that in too.

(On the other hand there may be use cases for *not* copying in credentials specifically?)

Reply:
I think, the argument against, is that might be a use-cases where were don't want to expose such things by default. When I had discussed exposing all of COSA to the VM, there was push back because the appliance was not special purpose.

Reply:
Isn't the appliance special purpose?  It's custom built dynamically to build CoreOS-style systems right?